lib/classes/CLI.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 const version = require('../../package.json').version;
+const enterpriseVersion = require('@serverless/enterprise-plugin/package.json').version;
```

Comment:
@medikoo I did the same in the past. Is this the recommended way to get access to the `package.json`?

Reply:
Yes, it's totally fine imo, just path could be shorter (`@serverless/enterprise-plugin/package` will work) but that's a nit

Reply:
Makes sense. I'd suggest to leave it for now since we're also requiring `package.json` above and keeping it consistent makes it easier to reason about.